### PR TITLE
installer: update time estimates

### PR DESCRIPTION
**Pull request by tomaskrizek**
Time estimates have been updated to be more accurate. Only
tasks that are estimated to take longer than 10 seconds have
the estimate displayed.

https://pagure.io/freeipa/issue/6596

### CODE REVIEW

**Comment by stlaz** [comment]
This will say:
```
Configuring certificate server (pki-tomcatd). Estimated time: 3 minutes
  [1/2]: configure certmonger for renewals
  [2/2]: Importing RA key
```
but the operation lasts ~5 seconds at most.

File: ipaserver/install/krainstance.py [hunk 1]
```diff
@@ -134,8 +134,13 @@ def configure_instance(self, realm_name, host_name, dm_password,
 
             self.step("enabling KRA instance", self.__enable_instance)
 
+        if ra_only:
```
Comment (tomaskrizek) [inline ipaserver/install/krainstance.py]: Not sure if this is necessary. Is kra install ever executed with `ra_only=True` option?

**Comment by tomaskrizek** [comment]
@stlaz That estimate was a bit off :) Thanks for noticing!

File: ipaserver/install/krainstance.py (hunk 1 above)
Comment (stlaz) [inline ipaserver/install/krainstance.py]: Please rebase on current master

File: ipaserver/install/service.py [hunk 2]
```diff
@@ -401,7 +401,7 @@ def start_creation(self, start_message=None, end_message=None,
                 else:
                     end_message = "Done configuring %s." % self.service_desc
 
-        if runtime > 0:
+        if runtime and runtime > 0:
```
Comment (stlaz) [inline ipaserver/install/service.py]: The correct condition is `if runtime is not None and and runtime > 0`

**Comment by stlaz** [comment]
Please fix the little issue and we will push this.

File: ipaserver/install/service.py (hunk 2 above)
Comment (MartinBasti) [inline ipaserver/install/service.py]: `SyntaxError: invalid syntax` :-P

Comment (stlaz) [inline ipaserver/install/service.py]: 🤕

**Comment by MartinBasti** [comment]
master:

* 09c6b7578046fed0824fc0f0d9040be69c0f0eb6 installer: update time estimates

**Pull request closed without merging**